ct/paperless-ngx.sh

```diff
@@ -53,6 +59,11 @@ function update_script() {
       cd /opt/paperless/src
       $STD uv run -- python manage.py migrate
       msg_ok "Updated Paperless-ngx"
+
+      if [[ -d /opt/paperless/backup ]]; then
```

Comment:
Why do you test for existence of the backup dir since you created it with mkdir -p before. It will be there.